cmd/prometheus/main.go

```diff
@@ -186,13 +186,16 @@ func (c *flagConfig) setFeatureListOptions(logger log.Logger) error {
 				level.Info(logger).Log("msg", "Experimental memory snapshot on shutdown enabled")
 			case "extra-scrape-metrics":
 				c.scrape.ExtraMetrics = true
-				level.Info(logger).Log("msg", "Experimental additional scrape metrics")
+				level.Info(logger).Log("msg", "Experimental additional scrape metrics enabled")
 			case "new-service-discovery-manager":
 				c.enableNewSDManager = true
 				level.Info(logger).Log("msg", "Experimental service discovery manager")
 			case "agent":
 				agentMode = true
 				level.Info(logger).Log("msg", "Experimental agent mode enabled.")
+			case "no-default-scrape-port":
+				c.scrape.NoDefaultPort = true
+				level.Info(logger).Log("msg", "Experimental no default scrape port enabled")
```

Comment:
Can we find a better wording?

Reply:
Sure. How about this: `"No default port will be appended to scrape targets' addresses."`